lib/ethpy/ethpy/hyperdrive/state/conversions.py

```diff
@@ -2,19 +2,6 @@
 from __future__ import annotations
 
 import re
-from dataclasses import asdict
-from typing import Any
-
-from fixedpointmath import FixedPoint
-from hypertypes.IHyperdriveTypes import Checkpoint as HtCheckpoint
-from hypertypes.IHyperdriveTypes import Fees as HtFees
-from hypertypes.IHyperdriveTypes import PoolConfig as HtPoolConfig
-from hypertypes.IHyperdriveTypes import PoolInfo as HtPoolInfo
-
-from .checkpoint import Checkpoint
-from .fees import Fees
-from .pool_config import PoolConfig
-from .pool_info import PoolInfo
 
 
 def camel_to_snake(snake_string: str) -> str:
```

Comment:
`camel_to_snake` is only used here, `ethpy/hyperdrive/addresses.py` and `chainsync/db/hyperdrive/convert_data.py`. `snake_to_camel` is only used here. So I think it makes sense for addresses.py and convert_data.py to use the new `hypertypes` functions and then you can delete this file entirely.

Reply:
yeah camel_to_snake kinda stuff should be in a top level place.  actually maybe in like ethpy/utilities/conversions.py or something since that would be a common thing people would do when working with smart contracts.